brevbaker-dsl/src/main/kotlin/no/nav/pensjon/brev/template/vedlegg/PDFTemplate.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.pensjon.brev.template.vedlegg
+
+import no.nav.pensjon.brev.template.Expression
+import no.nav.pensjon.brev.template.ExpressionScope
+import no.nav.pensjon.brev.template.LanguageSupport
+import no.nav.pensjon.brev.template.StableHash
+import no.nav.pensjon.brevbaker.api.model.PDFVedleggData
+import java.util.Objects
+
+class PDFTemplate<out Lang : LanguageSupport, AttachmentData : PDFVedleggData>(
```

Comment:
For å få stilen til å være enda nærmere slik vi gjør det for `LetterTemplate` og `AttachmentTemplate` så foreslår jeg å gjøre dette om til et interface/abstract class.
```
(abstract class / interface) PDFTemplate<out Lang : LanguageSupport, AttachmentData : PDFVedleggData> {
  fun template(data: AttachmentData): PDFVedlegg
  
  fun createVedlegg(scope: ExpressionScope<*>): PDFVedlegg = template(data.eval(scope))
}
```
Og da kan vi ha en hjelpemetode slik som for vanlige vedlegg:
```
fun createAttachmentPDF<Lang: LanguageSupport, AttachmentData : PDFVedleggData>(init: PDFVedlegg<Lang, AttachmentData>.() -> Unit): PDFTemplate<Lang, AttachmentData>
```

Reply:
Da vil man deklarere vedlegg med:
`val p1Vedlegg = createAttachmentPDF { /* ... */ }`

og bruke det med:
```
// ...
includeAttachmentPDF(pesysData.vedlegg, p1Vedlegg)
// ...
```